install/Docker/dockerfiles/scripts/build_odim.sh

```diff
@@ -17,6 +17,9 @@ LIST=`ls | grep -v 'lib-rest-client' | grep -E '^svc-|^plugin-|add-hosts'`
 echo $LIST
 flag=0
 for i in $LIST; do
+    if [[ "$i" == "svc-composition-service" ]]; then
```

Comment:
The if condition can be removed as there is no change to 'Dockerfile.odim' and svc-composition-service will not be listed here.